reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+
+However, they never make this explicit, and I think this is to the detriment of the paper.
+
+> We added a new section that deals with the concept of severity, making it explicit how our approach differs fundamentally.
+
+The fact that they subsequently add a Bayesian sauce to the likelihood ratio (by adding a distribution to the Type 1 error rate when a study is not preregistered, instead of simply making some assumption about a point-value of the inflated Type 1 error rate) does not add anything to their argument.
+So, I believe the authors have basically attempted to re-invent Mayo's severity argument.
+
+> The whole second part of the paper deals with the problem of not being able to reduce the theoretical risk to a point estimate.
+> The statistical procedure has an objective Type 1 error rate, for which we can come up with a point estimate if we agree on the conditions, which is a starting point for evaluating the theoretical risk of the scientific (!= statistical) procedure.
+> Since, a reader does not have access to the whole scientific procedure, they must judge it with uncertainty.
+> We find it important to distentengle the statistical procedure, the scientific procedure, and the outside perspective of the reader.
+
+The fact that this is not immediately clear is mainly due to missing discussions of the existing literature.
+Point 1: Preregistration can not objectively minimize uncertainty about the Type 1 error rate The main argument of the authors is that risky predictions create persuasive evidence if they turn out to be correct.
+Preregistrations are ideal for communicating the theoretical risk researchers took when testing a claim, and communicating this removes uncertainty in the reader about the theoretical risk the authors took.
+I think this argument is in the right direction, but it is not fully developed.
+I was somewhat disappointed not to see a discussion of my own paper on a conceptual analysis of the value of preregistration, as I had a similar goal as the authors when I wrote my paper, and my arguments are in the same direction, but dare I say, slightly better developed.
+
+> We find ourselves in a curios position, that perhaps may be summarised as "doing the right thing, for the wrong reasons", i.e., we agree with Laken's conclusions about preregistration but we disagree how to come to these conclusions.
+> We take a Bayesian/inductionist view, whereas Lakens takes a frequentist/anti-inductionist view, agreeing on the end result but disagreeing on everything that comes before.
+> While we find it important that readers are aware, how anti-inductionist deal with preregistration, this paper was not meant as an exercise to compare Bayesian justification for preregistration with its frequentist counterpart.
```

Comment:
At the same time, I feel it is important to compare these views and as a consequence of this comment, the manuscript should, in the discussion, address this point, that is, these two points of view and cite Laken's work.